src/Microsoft.DotNet.Interactive/KernelScheduler.cs

```diff
@@ -133,7 +134,7 @@ private void Run(ScheduledOperation operation)
     {
         _currentTopLevelOperation ??= operation.Value;
 
-        using var logOp = Log.OnEnterAndConfirmOnExit($"Run: {operation.Value}");
+        using var logOp = Log.OnEnterAndConfirmOnExit($"Run: {operation.Value.ToDisplayString("text/log")}");
```

Comment:
I'd suggest creating a `const` for this somewhere. We might quibble over the MIME type later and it will make it easier to change.

Reply:
I considered adding a `LoggingFormatter.MimeType` const. However, in order to use it across all the projects where this is currently referenced, we would need to make it public. It seems pretty harmless, but I wasn't sure about adding to the public API for our internal logging... I will go ahead and add it.

Reply:
We shouldn't make it public. 

Some of our logging infrastructure code is via shared files that are directly source included and this is a good fit for that.